client/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+name = "birdseed_client"
```

Should we change this to cashu_escrow_client?
```suggestion
name = "cashu_escrow_client"
```